espressif/vscode-esp-idf-extension
# update application insights ID

## Description

Update the Application Insights ID to point to the new subscription.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

Build the extension from this branch, run it. Verify that some records appear in Azure console.

## How has this been tested?

I have built the VSIX package, installed it, ran a few actions with IDF extension. Have checked that some events have appeared in Azure console:

<details><summary>(log)</summary>

timestamp [UTC] | name | client_OS | client_CountryOrRegion
-- | -- | -- | --
10/30/2023, 4:04:12.057 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:06:15.584 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:04:11.857 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:06:53.861 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:08:21.661 PM | espressif.esp-idf-extension/command | Linux 6.2.0-1015-azure | United States
10/30/2023, 4:08:21.730 PM | espressif.esp-idf-extension/command | Linux 6.2.0-1015-azure | United States
10/30/2023, 4:10:17.109 PM | espressif.esp-idf-extension/command | Linux 6.2.0-1015-azure | United States
10/30/2023, 4:10:19.523 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:10:30.039 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:11:23.776 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:11:33.210 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:04:12.280 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:05:56.127 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia
10/30/2023, 4:07:35.929 PM | espressif.esp-idf-extension/command | Darwin 22.6.0 | Czechia

</details>


## Dependent components impacted by this PR:

None

## Checklist
